go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/cosmos/ledger-go v0.9.2 // indirect
 	github.com/gogo/protobuf v1.3.3
 	github.com/golang/protobuf v1.4.3
-	github.com/google/renameio v1.0.0
+	github.com/google/renameio v0.1.0
```

Comment:
keep 1.0.0?

Reply:
ok, i will go for option2, change version , too.